ngraph/core/reference/include/ngraph/runtime/reference/ctc_greedy_decoder_seq_len.hpp

```diff
@@ -54,7 +54,7 @@ namespace ngraph
                         auto class_max_element =
                             std::max_element(class_index, class_index + class_count);
                         const auto max_class_ind = std::distance(class_index, class_max_element);
-                        if (max_class_ind < blank_index[0] &&
+                        if (max_class_ind != blank_index[0] &&
```

Comment:
I have a question about second output. Shouldn't it store the real size of decoded string rather than max size?
`out2[batch_ind] = out_index - batch_ind * seq_len_max;`

Reply:
Fixed